mk/docker/Dockerfile

```diff
@@ -32,19 +32,17 @@ ENV LC_ALL C.UTF-8
 ENTRYPOINT ["/bin/bash"]
 
 ### **** **** Add Non-CI Images Here **** **** ###
-## Example docker container with F prime code copied in:
-#FROM fprime-base AS fprime-docker
-#    COPY . /opt/fprime
-#    RUN python3 -m venv /opt/fprime-venv/ && . /opt/fprime-venv/bin/activate && \
-#        pip install wheel && \
-#        pip install /opt/fprime/Fw/Python/ && \
-#        pip install /opt/fprime/Gds/ && \
-#        rm -r ~/.cache/pip && \
-#        chown -R fprime:fprime /opt/fprime-venv && \
-#        chmod -R 775 /opt/fprime-venv
-#ENV VIRTUAL_ENV "/opt/fprime-venv"
-#ENV PYTHONHOME "/opt/fprime-venv"
-#ENV PATH "/opt/fprime-venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+FROM fprime-base AS fprime-docker
+RUN     git clone --quiet https://github.com/nasa/fprime.git /opt/fprime && \
+        python3 -m venv /opt/fprime-venv/ && . /opt/fprime-venv/bin/activate && \
+        pip install wheel && \
```

Comment:
we should make this `pip install -U wheel setuptools pip` as this is the latest F´ install recommendations.

Reply:
Specifically line 38 here, replace `pip install wheel` with the above.

Reply:
Addressed in bbce27c